instance/management/commands/instance_statistics_csv.py

```diff
@@ -0,0 +1,301 @@
+        if not settings.INSTANCE_LOGS_SERVER_HOST:
+            self.stderr.write(self.style.WARNING(
+                'Skipping Elasticsearch data collection because '
+                'INSTANCE_LOGS_SERVER_HOST is unset'
+            ))
+            return
+
+        inventory = '[apps]\n{server}'.format(server=settings.INSTANCE_LOGS_SERVER_HOST)
+        playbook_path = os.path.join(
+            settings.SITE_ROOT,
+            'playbooks/collect_instance_statistics/collect_elasticsearch_data.yml'
+        )
+
+        def log_line(line):
+            """Helper to pass to capture_playbook_output()."""
+            self.stderr.write(self.style.SUCCESS(line))
+        log_line.info = log_line
+        log_line.error = log_line
+
+        # Launch the collect_elasticsearch_data playbook, which places a file into the `playbook_output_dir`
+        # on this host.
+        ansible.capture_playbook_output(
+            requirements_path=os.path.join(
+                os.path.dirname(playbook_path),
+                'requirements.txt'
+            ),
+            inventory_str=inventory,
+            vars_str=(
+                'local_output_dir: {output_dir}\n'
+                'remote_output_filename: /tmp/activity_report\n'
+                'server_name_prefix: {server_name_prefix}\n'
+                'start_date: {start_date}\n'
+                'end_date: {end_date}'
+            ).format(
+                output_dir=playbook_output_dir,
+                server_name_prefix=name_prefix,
+                start_date=start_date,
+                end_date=end_date
+            ),
+            playbook_path=playbook_path,
+            username=settings.INSTANCE_LOGS_SERVER_SSH_USERNAME,
+            logger_=log_line,
+        )
+
+    def get_instance_usage_data(self, playbook_output_dir, name_prefix, public_ip):
+        """ Execute the collect_activity playbook to gather statistics """
+        inventory = '[apps]\n{server}'.format(server=public_ip)
+        playbook_path = os.path.join(
+            settings.SITE_ROOT,
+            'playbooks/collect_activity/collect_activity.yml'
+        )
+
+        def log_line(line):
+            """Helper to pass to capture_playbook_output()."""
+            self.stderr.write(self.style.SUCCESS(line))
+        log_line.info = log_line
+        log_line.error = log_line
+
+        # Launch the collect_activity playbook, which places a file into the `playbook_output_dir`
+        # on this host.
+        ansible.capture_playbook_output(
+            requirements_path=os.path.join(os.path.dirname(playbook_path), 'requirements.txt'),
+            inventory_str=inventory,
+            vars_str=(
+                'local_output_dir: {output_dir}\n'
+                'local_output_filename: user_statistics\n'
+                'remote_output_filename: /tmp/activity_report\n'
+                'config_section: {config_section}\n'
+                'extra_script_arguments: --skip-hit-statistics'
```

Comment:
@pcockwell, it will be useful to disable this flag when the `INSTANCE_LOGS_SERVER_HOST` is unset so that we at least get the hits data from the latest appserver.